content/docs/next-release/features.md

```diff
@@ -75,3 +75,7 @@ Also known as "Transitive Dependency Graph", where a chain `A -> B -> C` means t
 The node granularity of this graph can be changed between services and service endpoints. In the latter mode, different endpoints in the same service will be displayed as separate nodes, e.g. `A::op1` and `A::op2`.
 
 At this time the transitive graph can only be constructed from traces in the search results. In the future there will be a Flink job that will compute the graphs by aggregating all traces.
+
+## Aggregated Trace Metrics (ATM)
+
+Please refer to [Aggregated Trace Metrics (ATM)](../atm).
```

Comment:
I would give a one-sentence intro before directing to a link.

Reply:
NB: there is an intro page in the docs that also has a Features section, may want to include it there as wel